core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -63,20 +63,25 @@ private[spark] class CoarseMesosSchedulerBackend(
   // Maximum number of cores to acquire (TODO: we'll need more flexible controls here)
   val maxCores = conf.get("spark.cores.max",  Int.MaxValue.toString).toInt
 
+  val maxExecutorsPerSlave = conf.getInt("spark.mesos.coarse.executors.max", 1)
+  val maxCpusPerExecutor = conf.getInt("spark.mesos.coarse.cores.max", Int.MaxValue)
```

Comment:
the name of this value is very confusing. Say I'm a new user to Spark on Mesos and I see both `spark.cores.max` and `spark.mesos.coarse.cores.max`. Which one do I set, do I set both? Does one override the other?

I think a better name for this is probably `spark.mesos.coarse.coresPerExecutor`. It's a little more verbose, but there's no opportunity for ambiguity with this name.